ceph/rados/core_workflows.py

```diff
@@ -2142,20 +2142,36 @@ def get_inconsistent_object_details(self, pg_id):
             log.info(f"The inconsistent objects list is empty in the PG {pg_id}")
         return object_details
 
-    def check_inconsistent_health(self):
+    def check_inconsistent_health(self, inconsistent_present):
```

Comment:
We can have some default value to the newly added method param `inconsistent_present`, So that any existing method call does not fail.

Reply:
changes look good.